src/features/cleaner_file_type.rs

```diff
@@ -10,12 +10,16 @@ use super::utils::collect_metrics;
 
 pub fn directory_cleaner_based_on_file_type(
     dir: &String,
-    paths_to_clear: &[String],
+    types_to_clear: &[String],
     dry_run: bool,
     report_data: &mut ReportData,
+    paths_to_ignore: &[String],
```

Comment:
i think what we want here is files to ignore not paths to ignore in the sense that we don't want the user to have to type out the whole filePath - we want the to type just the file name. 


somthing like `dir-cleaner-rust - d <DIR> -s <SIZE> -i book.txt` not having to replace `book.txt `with the whole path